frontend/src/views/ComplianceReports/components/SummaryTable.jsx

```diff
@@ -195,7 +195,7 @@ const SummaryTable = ({
                       }}
                     >
                       {row?.format &&
-                      row[column.id] &&
+                      row[column.id] !== undefined &&
```

Comment:
question: Looks like this change is to enable rows with null ids to use the formatters. Why do we want to prevent formatting on undefined IDs but allow it on null?

Reply:
0 values come in as falsy so it never triggered the currency formatting ( this is what was preventing currency formatting for all cells that were currency cells ). we can probably change this to check for 0 values instead of a blanket statement of undefined. thoughts?


`(row[column.id] || row[column.id] === 0)`

Reply:
I like that a lot better, the 0 is much clearer for what we are trying to avoid. I thought it was for null but it was 0 all along.

Reply:
changed to `(row[column.id] || row[column.id] === 0)`